File: dev-superior-algoritmo-e-logica-de-programacao/c/exercicios/media_ponderada.c
#include <stdio.h>

int main() {
    int N;
    double a, b, c, media;

    printf("Quantos casos voce vai digitar? ");
    scanf("%d", &N);

    for (int i = 0; i < N; i++) {
        printf("Digite tres numeros:\n");
        scanf("%lf\n%lf\n%lf", &a, &b, &c);

        media = (a * 2.0 + b * 3.0 + c * 5.0) / 10.0;

        printf("MEDIA = %.1lf\n", media);
    }

    return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/media_ponderada.py
N: int
a: float; b: float; c: float; media: float

N = int(input("Quantos casos voce vai digitar? "))

for i in range(N):
	print("Digite tres numeros:")
	a = float(input())
	b = float(input())
	c = float(input())

	media = (a * 2.0 + b * 3.0 + c * 5.0) / 10.0

	print(f"MEDIA = {media:.1f}")